### PR TITLE
Add zulip links to the readme and index.

### PR DESCRIPTION
**Pull request by JoeZiminski**
This PR adds links to the Zulip chat to the README and index page. Please feel free to reword.

We also have the [discussions](https://github.com/neuroinformatics-unit/NeuroBlueprint/discussions) board for feedback, shall we pick one over the other? I am not very familiar with discussions, does it add anything over Zulip? Zulip seems nice as it more informal / chatty. But maybe discussions is a better record of discussions (or, we can chat on Zulip and have formal discussions on issues). I guess this question is relevant for feedback across @neuroinformatics-unit/neuroinformatics-team

### CODE REVIEW

**Comment by niksirbi** [comment]
I would definitely keep only one of the two. Having two places to congregate will split the community and be confusing.
Let's go all in on Zulip. I think discussions are not very Google-searchable.

**Comment by adamltyson** [comment]
TBH I forgot about discussions. I like zulip as it can be both:
- Real tech support
- General chat, planning collaborations

**Comment by adamltyson** [comment]
Interested to see what others think in general about this across NIU projects. Scrap discussions?

**Comment by alessandrofelder** [comment]
I find the napari zulip hard to parse/wrap my head around because I feel there's too many things going on at the same time (but maybe also that specific zulip instance is toooo big??? and I haven't made a concerted effort to customise to my needs...). I'd be more than happy to get rid of Discussions though, and we should definitely have this in only one place?

**Comment by adamltyson** [comment]
I'm not sure zulip is my favourite solution, but it's the only one I found that meets all our requirements. They're also kind enough to give us free cloud hosting.

**Comment by JoeZiminski** [comment]
I removed the discussions link from the text (not sure how to actually delete the discussions board). I reworded so please edit as necessary.

**Comment by JoeZiminski** [comment]
Also can anyone join that chat with the link, I see there is an option also to generate a temporary invite link but I guess this is only necessary if the room is private?

**Comment by adamltyson** [comment]
Yeah, I think we leave it open, and ideally make the main channels for each tool public (without sign in).

**Review by adamltyson — APPROVED** [review]
Seems like we've discussed everything there is to discuss here. Going to merge this and disable discussions.
